src/com/centeksoftware/parclock/javarelay/MainFrame.java

```diff
@@ -247,6 +268,23 @@ private void initComponents()
 		action1 = new javax.swing.JLabel();
 		action2 = new javax.swing.JLabel();
 		
+		SimpleDateFormat sdfDate = new SimpleDateFormat("yyyy-MM-dd_HH;mm;ss");// dd/MM/yyyy
+		Date now = new Date();
+		String timestamp = sdfDate.format(now);
+		timestamp = timestamp.replace(" ", "_");
+		
+		filename = "C:\\Users\\Ian\\Documents\\log_" + timestamp + ".txt"; // replace "C:\\Users\\Ian\\Documents\\" with actual file location
```

Comment:
Replace the file path with `"./logs/log_" + timestamp _ ".txt"`. This will place it in a "logs" folder in the directory `relay.jar` is run from. Never use absolute paths unless necessary, and definitely don't make the end user recompile the program to change a setting. Also, use `/` instead of `\` for file paths because the latter is only compatible in windows while the former works in linux, mac, and windows.